src/components/FlatmapVuer.vue

```diff
@@ -1657,13 +1673,13 @@ export default {
     },
     // checkNeuronClicked shows a neuron path pop up if a path was recently clicked
     createConnectivityBody: function () {
-      if (Object.keys(this.relevanceEntry).length > 0) {        
-        const featureIds = Object.values(this.relevanceEntry)
+      if (Object.keys(this.connectionEntry).length > 0) {
+        const featureIds = Object.values(this.connectionEntry)
         const body = {
           type: 'connectivity',
-          sourceId: featureIds[0],
-          targetId: featureIds[featureIds.length - 1],
-          intermediateIds: [],
+          source: featureIds[0],
+          target: featureIds[featureIds.length - 1],
+          intermediates: [],
```

Comment:
Line 1686 needs to be updated as well. `body.intermediateIds.push(id)` to `body.intermediates.push(id)`